src/figanos/matplotlib/plot.py

```diff
@@ -1409,21 +1432,60 @@ def heatmap(
     plot_kw.setdefault("cmap", cmap)
 
     # plot
-    sns.heatmap(df, ax=ax, **plot_kw)
-
-    # format
-    plt.xticks(rotation=45, ha="right", rotation_mode="anchor")
-    ax.tick_params(axis="both", direction="out")
+    def draw_heatmap(*args, **kwargs):
+        data = kwargs.pop("data")
+        d = (
+            data
+            if len(args) == 0
+            # Any sorting should be performed before sending a DataArray in `fg.heatmap`
+            else data.pivot_table(
+                index=args[1], columns=args[0], values=args[2], sort=False
+            )
```

Comment:
Columns/Rows of individual heatmap are not sorted by `pandas` in this way. This should be managed by sorting directly the input dataset, I think it's reasonable. Otherwise, it's difficult to manage custom order of coords values in the heatmap.